test/unit/user.controller.test.js

```diff
@@ -0,0 +1,394 @@
+var assert = require("assert");
+const userController = require('../../controller/user/user.controller');
+const monk = require('monk');
+require('mocha-sinon');
+const sinon = require('sinon');
+const { expect, use } = require("chai");
+const { doesNotMatch, fail } = require("assert");
+let db = monk('localhost:27017/brainbox');
+
+describe('User Controller: ', function () {
+    describe('validator function() ', function () {
+        it('should perform the validations correctly', async function () {
+            let req = {};
+            let res = {
+                send: sinon.spy()
+            };
+            await userController.validator(req, res, () => { });
+            assert.strictEqual(res.send.callCount, 0);
+            sinon.restore();
+        });
+    });
+
+    describe('user function() ', function () {
+        it('should return the requested user with valid input', async function () {
+            let req = {
+                user: {
+                    username: 'anyone'
```

Comment:
Brainbox uses "nickname" for "username". Please check the "user" collection in the Mongo DB

Reply:
Resolved..